modules/deploy/pages/deployment-option/self-hosted/kubernetes/kubernetes-cluster-requirements.adoc

```diff
@@ -82,20 +194,39 @@ Redpanda uses the following default ports:
 | Admin API
 |===
 
-Minimum 10 GigE.
+To change these ports, see xref:manage:kubernetes/networking/configure-listeners.adoc[].
 
-Redpanda recommends using NodePorts instead of Loadbalancers. See the xref:./kubernetes-best-practices.adoc#external-networking[external networking best practices].
+Depending on your deployment and security policies, you may not be able to access worker nodes through a NodePort Service.
+If you choose to use another Service, consider the impact on the cost and performance of your deployment:
 
-== Tuning
+* LoadBalancer Service - To make each Redpanda broker accessible with LoadBalancer Services, you need one LoadBalancer Service for each Redpanda broker so that requests can be routed to specific brokers rather than balancing requests across all brokers. Load balancers are expensive, add latency and occasional packet loss, and add an unnecessary layer of complexity.
+* Ingress - To make each Redpanda broker accessible with Ingress, you need to run an Ingress controller and set up routing to each Redpanda broker. Routing adds latency and can be a throughput bottleneck.
```

Comment:
Ingress - To make each Redpanda broker accessible with Ingress, you must run an Ingress controller and set up routing to each Redpanda broker. Routing adds latency and can be a throughput bottleneck.